apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -1,5 +1,8 @@
 package com.linkedin.openhouse.jobs.client;
 
+import static org.apache.iceberg.TableProperties.WRITE_TARGET_FILE_SIZE_BYTES;
```

Comment:
Not sure if we should reuse WRITE_TARGET_FILE_SIZE_BYTES. Thinking of this as a config used at the time of write which might be tuned to make writes faster as it depends upon what the writer wants to trade off.

Other two options are:

1. Compute the best target file size bytes [preferred]
2. Add a separate policy knob to tune [less ideal, as we dont want end-users to pay the tax of knowing what is the idea value for the config]
3. Start with a constant value and then work towards [1] [short-term]

Reply:
From a separation of concerns standpoint, agree with option (2). Also adding @anjagruenheid to get her thoughts on this. This is something we should account for when proposing the right OptimizationStrategy for the table.

For short-term (for context of this PR), let's start with a constant compaction job setting `RewriteDataFiles.TARGET_FILE_SIZE_BYTES` so that OpenHouse can control that value.

Reply:
I think it makes sense to start with an internal compaction job and to avoid exposing these variables to the user. File size specifically is mostly beneficial for the engine itself (i.e., maximize parallelization by making sure that the number of files is equivalent to the numbers of executors, or to avoid limitations such as memory/executor node), thus it makes sense to keep this internal.

I like the idea of a periodic job that calculates this parameter because that would also give you some of the infrastructure to calculate potential compaction trigger points or the cost/benefit analysis that we had talked about based on table statistics.

Reply:
Thank you for the input. If I can summarize, we don't want to use the table property `TableProperties.WRITE_TARGET_FILE_SIZE_BYTES` since we want to decouple writer config from compaction config. I didn't want to hard-code the target size in scheduler since we already have tables that need non-default value. So, ideally until we have introduced compaction table properties, I'd propose to use current approach since we're going to read in a similar way another table property.

Reply:
To make this PR simpler, after taking to @sumedhsakdeo, I'm going to remove any arguments from DC task and rely on https://github.com/apache/iceberg/blob/e340ad5be04e902398c576f431810c3dfa4fe717/core/src/main/java/org/apache/iceberg/actions/BinPackStrategy.java#L139, we can design how to store rewrite strategy separately